src/templates/Practice3.tsx

```diff
@@ -0,0 +1,10 @@
+import Input from "../components/Input";
+
+export default function Practice2()
+{
+    return (
+        <div className='flex flex-col items-center mt-14 mb-50 mx-96...'>
+        <Input></Input>
```

Comment:
Just a self-closing tag should be fine since we didn't put any children in it.

```suggestion
        <Input/>
```

Reply:
aah sorry didn't notice, made the changes